src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_TripSearchScreen/components/ResultRow.tsx

```diff
@@ -151,7 +144,7 @@ const tripSummary = (
       ),
     ) + screenReaderPause;
   const passedTripText = isInPast
-    ? t(TripSearchTexts.results.resultItem.passedTrip)
+    ? t(TripSearchTexts.results.resultItem.passedTrip) + ', '
```

Comment:
Will the comma always be present when using this translation? If so, could it be a part of the translation?

Reply:
Cool, all good then